addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/SessionDetectionScanRule.java

```diff
@@ -85,7 +85,7 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
             // The request is using at least one session token, do we know where they come from?
             List<SessionToken> foundTokens = new ArrayList<>();
             for (SessionToken st : requestTokens) {
-                SessionToken sourceToken = AuthUtils.containsSessionToken(st.getToken());
+                SessionToken sourceToken = AuthUtils.containsSessionToken(st.getValue());
```

Comment:
FYI this is a bug fix!

Reply:
Should it be mentioned in the changelog?

Reply:
The bug would not have been visible to users - the following code will still have found the token, it will just have taken longer than it should 😁